### DISCUSSION TITLE
Easier way to browse

### DISCUSSION THREAD

**Discussion by stevealexrs** [SunWeb3Sec/DeFiHackLabs#251] [General]
I enjoy reading these compiled write-ups but it can be difficult to navigate. I need to scroll down and click on the unhide button to search for it. It is even worse on mobile. Any idea how to make it easier? Maybe separate page for different years? A downloadable version? 📖

The link to specific GitHub section in notion is not clickable.

**Comment by SunWeb3Sec** [comment]
Thanks for the feedback. We know the issue. We will try to solve it once get some time.

**Comment by SunWeb3Sec** [comment]
@stevealexrs how about this?
https://github.com/SunWeb3Sec/DeFiHackLabs/tree/main/past
